packages/ui-patterns/src/McpUrlBuilder/components/ClientSelectDropdown.tsx

```diff
@@ -77,7 +77,7 @@ export const ClientSelectDropdown = ({
           </Button>
         </PopoverTrigger_Shadcn_>
       </div>
-      <PopoverContent_Shadcn_ className="mt-0 p-0 max-w-48" side="bottom" align="start">
+      <PopoverContent_Shadcn_ className="mt-0 p-0 max-w-48" side="bottom" align="start" portal>
```

Comment:
`portal` was the key to fixing the scroll weirdness, should probably use this anywhere we use popovers in the future